projects/core/src/lib/backend/backend.service.ts

```diff
@@ -38,15 +38,36 @@ import {
     LayerDict,
     LayerCollectionDict,
     AuthCodeRequestURL,
+    BackendInfoDict,
 } from './backend.model';
 
+export interface BackendStatus {
+    available: boolean;
+    httpError?: HttpErrorResponse;
+    initial?: boolean;
```

Comment:
is this ever set to false?

Reply:
yes initial state is false and it might be needed when we do real backend monitoring